.github/workflows/release.yml

```diff
@@ -9,6 +9,9 @@ on:
 jobs:
   release:
     if: |-
-      ${{ startsWith(github.event.head_commit.message, 'Release: v') }}
+      startsWith(github.event.head_commit.message, 'Release: v')
```

Comment:
Don't remove this please - using the `${{` makes the syntax highlighting and vscode plugin more functional